torch/distributed/nn/api/remote_module.py

```diff
@@ -33,6 +33,14 @@
     instantiator.instantiate_non_scriptable_remote_module_template()
 )
 
+_REMOTE_MODULE_PICKLED_ATTRIBUTES = (
```

Comment:
We should also update the docs of RemoteModule mentioning that we can share it across processes using the RPC framework without actually incurring any copy overhead of the actual module and it behaves like an RRef pointing to the remote module.

Reply:
Added to the docstring.